### PR TITLE
Support recent upstream changes

### PR DESCRIPTION
**Pull request by pdf**
Bump server version to v-2017-03-27

### CODE REVIEW

**Comment by pdf** [comment]
Updated the test script with a valid asset path to allow the build to pass.  Out of curiosity, is there a reason for compiling assets on every startup rather than as a docker build step?

**Comment by jsturgis** [comment]
Thanks for the PR, I am working on running compile_resources.sh at build time instead of at runtime.

**Comment by pdf** [comment]
Since it calls perl, it probably needs `eval $( perl -Mlocal::lib)` to populate the env before running it.  The method I used to run `cpanm` in the Dockerfile appears to do the job.

Would you like me to update this PR with that change?

File: musicbrainz-dockerfile/DBDefs.pm [hunk 1]
```diff
@@ -52,16 +52,15 @@ MusicBrainz::Server::DatabaseConnectionFactory->register_databases(
...
     READWRITE => {
         database    => "musicbrainz_db",
-        schema          => "musicbrainz",
+        schema      => "musicbrainz",
         username    => "$ENV{DB_ENV_POSTGRES_USER}",
-        password        => "$ENV{DB_ENV_POSTGRES_PASSWORD}",
-        host            => "$ENV{DB_PORT_5432_TCP_ADDR}",
-        port            => "$ENV{DB_PORT_5432_TCP_PORT}",
+        password    => "$ENV{DB_ENV_POSTGRES_PASSWORD}",
+        host        => "db",
```
Comment (jsturgis) [inline musicbrainz-dockerfile/DBDefs.pm:58]: is there a reason that you are using a hardcoded host and port value instead of using the env variables?

**Review by jsturgis — CHANGES_REQUESTED** [review]

File: musicbrainz-dockerfile/DBDefs.pm (hunk 1 above)
Comment (pdf) [reply musicbrainz-dockerfile/DBDefs.pm:58]: Use of env vars to connect to linked services is [deprecated](https://docs.docker.com/compose/link-env-deprecated/), and it's now recommended to just use the link name directly.

Comment (jsturgis) [inline musicbrainz-dockerfile/DBDefs.pm]: I see thanks for the info, is there a doc you can point me to with more details on this?

Comment (jsturgis) [reply musicbrainz-dockerfile/DBDefs.pm:58]: ok cool thanks for the info.

**Review by jsturgis — APPROVED** [review]

**Pull request merged**